src/sdam/server.ts

```diff
@@ -273,7 +273,7 @@ export class Server extends TypedEventEmitter<ServerEvents> {
   public async command(
     ns: MongoDBNamespace,
     command: Document,
-    options?: CommandOptions
+    options: CommandOptions
```

Comment:
there are no instances in the driver where we don't pass options into this method